earth_enterprise/src/common/tests/SConscript

```diff
@@ -50,8 +50,16 @@ map(lambda test: env.test(test, [test + '.cc'],
     ['etencoder_tests',
      ])
 
+testdir = os.path.join(env.exportdirs['bin'],'tests')
+env.copyfile(testdir, 'RunAllTests.pl')
 
-env.copyfile(os.path.join(env.exportdirs['bin'],'tests'),
-             'RunAllTests.pl')
+# Drop in the test data
+env.Command(
```

Comment:
Yeah, scons syntax is confusing.
My suggestion is combining all test setup actions into one command. So if anything else is required for test, it will be added to the list in the env.Command().

Why do we need to use/return anything from env.copyfile() in env.Command().

Source in env.Command() seems allow to set some dependencies:
Suggested above:
- specifies to run set of commands in the list [..] for target "install_for_test";
- specifies dependency on testdir: $SOURCE=testdir;
- creates .install_for_test file (TARGET) showing that the target "install_for_test" has been successful.